lib/ircs/plugins/topic_sync.rb

```diff
@@ -0,0 +1,66 @@
+# vim: fileencoding=utf-8
+
+require_relative 'base'
+
+module LogArchiver
+  module Plugin
+    # JOIN したチャンネルの TOPIC がデータベースに保存されているか確認する
+    class TopicSync < Base
+      include Cinch::Plugin
+
+      set(plugin_name: 'TopicSync')
+
+      listen_to(333, method: :get_topic_metadatas)
+
+      # チャンネルに JOIN したときの Numeric Reply 333 から、
+      # チャンネルが作成されたタイムスタンプと作成者を取得する
+      # @param [Cinch::Message] m
+      # @return [void]
+      def get_topic_metadatas(m)
```

Comment:
dataは複数形なので、metadataの方が自然だと思います。

```suggestion
      def get_topic_metadata(m)
```

Reply:
修正しました。